### PR TITLE
[#33] Create dummy sign in and sign up with supabase

### PR DESCRIPTION
**Pull request by Trowl3r**
### ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers
This is one of my first contribs to open source in generall, so I would really appreciate feedback! 
in the .env.template the values need to be replaced with the real superbase backend keys.
The implementation is really simple, no real styles, no handling of the user, no real error handling.
Idk if this is enough for the start, please just feedback me!.

### CODE REVIEW

**Comment by ykdojo** [comment]
Thank you so much!

I'm wondering if you could add some documentation on how to set it up with Supabase?

**Comment by rohitdasu** [comment]
@Trowl3r could you please check why vercel deployment failing? thanks

**Comment by Trowl3r** [comment]
> I'm wondering if you could add some documentation on how to set it up with Supabase?
@ykdojo Of course. I will add it in the ReadMe below the deploy to vercel part.

> @Trowl3r could you please check why vercel deployment failing? thanks
@rohitdasu I don't know how I can see why Vercel crashed, since I don't have access to anything. But I would assume it is because the utils/supabaseClient.ts needs the .env vars. But I am not sure. The rest ist just simple jsx, so I would assume that as the problem. But I am not sure.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,40 @@ yarn run build && npm run start (in case of Yarn, it will build for production a
...
+
+`#1` Sign up on Supabase [here](https://supabase.com/).
+
+`#2` Create a new Project inside Supabase
+
+`#3` Go to settings -> API and copy the Project ```URL``` and ```Anon/Public```
+
+`#4` Paste the values in the .env.template at ```NEXT_PUBLIC_SUPABASE_URL``` for the URL and ```NEXT_PUBLIC_SUPABASE_ANON_KEY``` for the anon.
+
+`#5` Go th the [SQL Editor](https://app.supabase.com/project/_/sql) inside the Supabase Dashboard
```
Comment (ykdojo) [inline README.md:70]: "Go to the"?

Comment (Trowl3r) [reply README.md:70]: Ah I forgot the Page ther Go the the SQL Editor Page inside the Supabase Dashboard was, what I intended to write

Comment (rohitdasu) [reply README.md:70]: good catch @ykdojo

File: README.md [hunk 2]
```diff
@@ -57,6 +57,40 @@ yarn run build && npm run start (in case of Yarn, it will build for production a
...
 https://defaang.vercel.app/
+
+
+
+
+
+
+
+
+
```
Comment (ykdojo) [inline README.md:89]: what are all these blank lines for? I just noticed

**Comment by ykdojo** [comment]
A few more things:

- When a user signs up at /signup, confirming the email is required. I think we should make that clear.

**Comment by ykdojo** [comment]
There are a few remaining issues, but I think I'll merge this one as is anyway. I'll create those issues as separate ones.

**Comment by Trowl3r** [comment]
Alright, I just wanted to implement the simple auth implementation. For any related issues I would like to work on them if I am fast enough, to improve more! Thanks!!

**Comment by rohitdasu** [comment]
@ykdojo i don’t think merging this PR without resolving the vercel deployment was a good idea. Now our new PR merges don't reflect on the production 😢

**Comment by ykdojo** [comment]
@rohitdasu actually I already resolved it! it should be okay now - did you check?

**Comment by rohitdasu** [comment]
> @rohitdasu actually I already resolved it! it should be okay now - did you check?

yep, it's fixed now. Thanks 😊

File: README.md (hunk 2 above)
Comment (ykdojo) [reply README.md]: (looks like they're not there in the current main)

**Pull request merged**